OCPBUGS-48234: added an xref to PriorityClass creation (#91045)
openshift/openshift-docs: modules/compliance-priorityclass.adoc

In large scale environments, the default `PriorityClass` object can be too low to guarantee Pods execute scans on time. For clusters that must maintain compliance or guarantee automated scanning, it is recommended to set the `PriorityClass` variable to ensure the Compliance Operator is always given priority in resource constrained situations.

.Prerequisites

> * Optional: Create a `PriorityClass` object. See _Configuring priority and preemption_ in the _Additional Resources_ section for more information.

Reviewer: This feels like a naive question, but is there a way to link directly? Or is that against convention?

Author: Sadly, that is against convention at this time. Long story short, linking from modules breaks the docs build or the webpage or both.

> * Optional: Create a `PriorityClass` object. See _Configuring priority and preemption_ in the _Additional Resources_ section for more information.

Reviewer 2: For more information, see "<section_name>".  Quotation must be used. "in the _Additional Resources_ section" can be dropped. see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentences

Reviewer 2: As per https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites, prerequisites are checks that are true or that the user must have completed before they begin a procedure. I see that the pre requisites are optional.

> * Optional: Create a `PriorityClass` object. See _Configuring priority and preemption_ in the _Additional Resources_ section for more information.

Reviewer 3: ```suggestion
* Optional: You have created a `PriorityClass` object. For more information, see "Configuring priority and preemption".
```

Reviewer 3: https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites
